spring-batch-docs/asciidoc/footer/index-footer.adoc

```diff
@@ -2,7 +2,7 @@
 Lucas Ward, Dave Syer, Thomas Risberg, Robert Kasanicky, Dan Garrette, Wayne Lund,
 Michael Minella, Chris Schaefer, Gunnar Hillert, Glenn Renfro, Jay Bryant, Mahmoud Ben Hassine
 
-Copyright © 2009, 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018 Pivotal, Inc. All Rights
+Copyright © 2009, 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019 Pivotal, Inc. All Rights
```

Comment:
You don't need to change it, but you can use "2009 - 2019" rather than have a long list.